warp10/src/main/java/io/warp10/script/mapper/MapperToDouble.java

```diff
@@ -58,8 +95,15 @@ public Object apply(Object[] args) throws WarpScriptException {
         value = Boolean.TRUE.equals(values[0]) ? 1.0D : 0.0D;
       } else if (values[0] instanceof String) {
         try {
-          value = Double.valueOf((String) values[0]);
-        } catch (NumberFormatException nfe) {
+          if (null == this.language) {
+            value = Double.valueOf((String) values[0]);
+          } else
+          {
+              Locale locale = Locale.forLanguageTag(this.language);
```

Comment:
This is inefficient, you are doing it in every call to apply, move it to the constructor and set final elements. Are their use thread safe? If not it should be specified in the documentation now that CEVAL exists.